src/components/SupportPanel.vue

```diff
@@ -0,0 +1,185 @@
+    const speakClicked = ref(false);
+    const moneyClicked = ref(false);
+    const helmetClicked = ref(false);
+    const brainClicked = ref(false);
+
+    const toggleHeartIcon = () => {
+      heartClicked.value = !heartClicked.value;
+    };
+    const toggleBrainIcon = () => {
+      brainClicked.value = !brainClicked.value;
+    };
+    const toggleSpeakIcon = () => {
+      speakClicked.value = !speakClicked.value;
+    };
+    const toggleMoneyIcon = () => {
+      moneyClicked.value = !moneyClicked.value;
+    };
+    const toggleHelmetIcon = () => {
+      helmetClicked.value = !helmetClicked.value;
+    };
+
+    const alteri = computed(() => store.state.nwk.alteri);
+
+    const alteriEmotional = computed(() => {
+      const emotionalAlteri = store.state.nwk.alteri.filter(
+        (alter) => alter.supportEmotional >= 1 && alter.supportEmotional <= 3
+      );
+      return emotionalAlteri.map((alter) => alter.name).join(", ");
+    });
+
+    const alteriMaterial = computed(() => {
+      const emotionalAlteri = store.state.nwk.alteri.filter(
+        (alter) => alter.supportMaterial >= 1 && alter.supportMaterial <= 3
+      );
+      return emotionalAlteri.map((alter) => alter.name).join(", ");
+    });
+
+    const alteriCognitive = computed(() => {
+      const emotionalAlteri = store.state.nwk.alteri.filter(
+        (alter) => alter.supportCognitive >= 1 && alter.supportCognitive <= 3
+      );
+      return emotionalAlteri.map((alter) => alter.name).join(", ");
+    });
+
+    const alteriPractical = computed(() => {
+      const emotionalAlteri = store.state.nwk.alteri.filter(
+        (alter) => alter.supportPractical >= 1 && alter.supportPractical <= 3
+      );
+      return emotionalAlteri.map((alter) => alter.name).join(", ");
+    });
+
+    const alteriSocial = computed(() => {
+      const emotionalAlteri = store.state.nwk.alteri.filter(
+        (alter) => alter.supportSocial >= 1 && alter.supportSocial <= 3
+      );
+      return emotionalAlteri.map((alter) => alter.name).join(", ");
+    });
+
+    return {
+      isOpen: isOpen,
+      heartClicked: heartClicked,
+      brainClicked: brainClicked,
+      moneyClicked: moneyClicked,
+      speakClicked: speakClicked,
+      helmetClicked: helmetClicked,
+      toggleHeartIcon,
+      toggleBrainIcon,
+      toggleSpeakIcon,
+      toggleMoneyIcon,
+      toggleHelmetIcon,
+      alteri: alteri,
+      alteriEmotional: alteriEmotional,
+      alteriCognitive: alteriCognitive,
+      alteriSocial: alteriSocial,
+      alteriMaterial: alteriMaterial,
+      alteriPractical: alteriPractical,
+      toggleEmotional: () => store.commit("session/toggle", "emotional"),
+      toggleCognitive: () => store.commit("session/toggle", "cognitive"),
+      toggleSocial: () => store.commit("session/toggle", "social"),
+      toggleMaterial: () => store.commit("session/toggle", "material"),
+      togglePractical: () => store.commit("session/toggle", "practical"),
+    };
```

Comment:
vereinfachen: state nur in session store speichern
statt den ref's ein computed aus der session
toggleIcon ist dann nicht mehr nötig